From 1f37a773ffec72e917a759b5bdd6ae834abaa36c Mon Sep 17 00:00:00 2001
From: Derek Cannata <thederekjay@gmail.com>
Date: Thu, 10 Jan 2013 17:51:33 -0500
Subject: [PATCH 12/13] Fix DSP Manager

Not sure what happened here? Need CM bits for DSP to work properly..

Change-Id: Ia6efc2514a45c6adb55b91cf61ac0ae8ac25c6e5
---
 audio_effects.conf | 25 +++++++++++++++++++------
 1 file changed, 19 insertions(+), 6 deletions(-)

diff --git a/audio_effects.conf b/audio_effects.conf
index d681c69..0399b0a 100644
--- a/audio_effects.conf
+++ b/audio_effects.conf
@@ -9,6 +9,9 @@ libraries {
   bundle {
     path /system/lib/soundfx/libbundlewrapper.so
   }
+  cm {
+    path /system/lib/soundfx/libcyanogen-dsp.so
+  }
   reverb {
     path /system/lib/soundfx/libreverbwrapper.so
   }
@@ -39,17 +42,27 @@ libraries {
 #    }
 
 effects {
+  compression {
+    library cm
+    uuid f27317f4-c984-4de6-9a90-545759495bf2
+  }
   bassboost {
-    library bundle
-    uuid 8631f300-72e2-11df-b57e-0002a5d5c51b
+#    library bundle
+#    uuid 8631f300-72e2-11df-b57e-0002a5d5c51b
+    library cm
+    uuid 42b5cbf5-4dd8-4e79-a5fb-cceb2cb54e13
   }
   virtualizer {
-    library bundle
-    uuid 1d4033c0-8557-11df-9f2d-0002a5d5c51b
+#    library bundle
+#    uuid 1d4033c0-8557-11df-9f2d-0002a5d5c51b
+    library cm
+    uuid 7c6cc5f8-6f34-4449-a282-bed84f1a5b5a
   }
   equalizer {
-    library bundle
-    uuid ce772f20-847d-11df-bb17-0002a5d5c51b
+#    library bundle
+#    uuid ce772f20-847d-11df-bb17-0002a5d5c51b
+    library cm
+    uuid 58bc9000-0d7f-462e-90d2-035eddd8b434
   }
   volume {
     library bundle
-- 
1.8.1.1

